ghga_transpiler/config/config.py

```diff
@@ -43,6 +44,7 @@ class WorksheetSettings(BaseModel):
     start_row: Optional[int]
     start_column: Optional[int]
     end_column: Optional[int]
+    transformations: dict[str, Callable] = {}
```

Comment:
```suggestion
    transformations: Optional[dict[str, Callable]]
```
This needs to default to `None` so that this
https://github.com/ghga-de/ghga-transpiler/blob/e978e486776b9dc7d98a1364edc1f5a42f371413/ghga_transpiler/config/config.py#L67-L69
works, right?